src/lib/structures/Command.ts

```diff
@@ -160,6 +160,7 @@ export class Command {
 				this.run = options.run || Command.defaults?.run;
 				this.onError = options.onError || Command.defaults?.onError;
 				this.autoDefer = options.autoDefer || Command.defaults?.autoDefer;
+				this.options = options || Command.defaults?;
```

Comment:
You should merge the defaults with the options (options overwrites defaults). Be careful to not overwrite the defaults with the merged one.